src/Utils/cleanObject.ts

```diff
@@ -0,0 +1,22 @@
+const cleanObject = (object) => {
```

Comment:
It would be great if you can add a comment block on top to describe this function.

Reply:
@sarveshpro Done